apps/yggdrasil/lib/yggdrasil/user.ex

```diff
@@ -1,6 +1,7 @@
 defmodule Yggdrasil.User do
   use Ecto.Schema
   import Ecto.Changeset
+  import Ecto.Query, only: [from: 1, from: 2] # dislike this
```

Comment:
Why, i get firsts of anger if I keep it in :smile:. haha. I'll remove it, I tend to think out loud in comments for some reason and usually do remove them.